src/getting-started/create-web-app.md

```diff
@@ -0,0 +1,3 @@
+# Create Web App
+
+TODO: write about carton and Tokamak?
```

Comment:
@MaxDesiatov Could you add carton and Tokamak section? 🙏

Reply:
I'll submit those in a separate PR for review.

Reply:
OK, I'll merge this PR after your PR will be merged into this branch.

Reply:
This file can be deleted, the one with a quick introduction to Tokamak is named `browser-app.md`.